test-data/unit/check-abstract.test

```diff
@@ -326,7 +326,7 @@ class A(I, J):
     def f(self, x: str) -> int: pass \
         # E: Argument 1 of "f" is incompatible with supertype "I"; supertype defines the argument type as "int"
     def g(self, x: str) -> int: pass \
-        # E: Return type of "g" incompatible with supertype "J"
+        # E: Return type "int" of "g" incompatible with return type "str" of supertype "J"
```

Comment:
I think it is better to say:
```
Return type "int" of "g" incompatible with return type "str" in supertype "J"
```
i.e. replace `of` -> `in`.

Reply:
the `of` here refers to the function. I.e. the function "g" has a return type `of` "int".
Whereas the `in` that comes later refers to the superclass i.e. return type "str" of the function "g" `in` class "J".
Do you still think it makes sense to change the `in` to `of`? I haven't heard people refer to a return type `in` a function... (although I may be wrong!)

Reply:
you are totally correct - sorry I misread!

Reply:
There is a slight edge case where the function name doesn't match the superclass function name. You end up with this message:

`Return type "A" of "__iadd__" incompatible with return type "B" in "__add__" of supertype "A"` 

This due to how `override_target` is implemented. Do you think this is OK? Or should I try and handle it?

Reply:
> This due to how `override_target` is implemented. Do you think this is OK? Or should I try and handle it?

It is a bit suboptimal, but I wouldn't bother, it is already much better than it used to be.